app/sass/_base.scss

```diff
@@ -1,121 +1,155 @@
 body {
-  width: 90%;
-  background-color: $color-scheme8;
-  text-align: center;
   font-family: 'Fira Sans', sans-serif;
+  font-size: 16px;
 }
-.main {
-  @include flex-format;
-  top: 0;
-  bottom: 20px;
-  position: fixed;
-  overflow-y: scroll;
-  overflow-x: hidden;
+
+.wrapper {
```

Comment:
Move this wrapper class to appropriate file. This seems more like layout styles more than base